hugo/content/guides/builtin-library.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Builtin Libraries"
+weight: 300
+---
+
+Languages usually offer their users some high-level programming features that they do not have to define themselves.
+For example, TypeScript provides users with typings for globally accessible variables such as the `window`, `process` or `console` objects.
+They are part of the JavaScript runtime, and not defined by any user or a package they might import.
+Instead, these features are contributed through what we call a builtin library.
+
+Fortunately, loading a builtin library in Langium is very simple. We first start off with defining the source code of the library using the *hello world* language from the [getting started guide](/docs/getting-started):
+
+```ts
+export const builtinHelloWorld = `
+person Jane
+person John
+`.trimLeft();
+```
+
+Next, we load our builtin library code through the `loadAdditionalDocuments` provided by the `DefaultWorkspaceManager`:
+
+```ts
+import {
```

Comment:
Might be nice to clarify where this content should be located.

Reply:
The workspace manager is fairly isolated, so users can place it anywhere they like. I wouldn't impose a folder/directory structure for them in this simple guide.